src/devices/bus/sg1000_exp/sk1100prn.cpp

```diff
@@ -108,10 +108,10 @@ READ_LINE_MEMBER(sk1100_printer_port_device::fault_r)
 
 READ_LINE_MEMBER(sk1100_printer_port_device::busy_r)
 {
-	if (m_device)
-		return m_device->output_busy();
-	else
-		return 1;
+        if (m_device)
+                return m_device->output_busy();
+        else
+                return 1;
```

Comment:
Why did you change the indentation in these lines?
Keep the tabs for indentation everywhere in your pull request, please.